web-local/src/lib/components/navigation/models/ModelMenuItems.svelte

```diff
@@ -45,7 +65,43 @@
   // const metricMigrate = useRuntimeServicePutFileAndReconcile();
 
   /** functionality for bootstrapping a dashboard */
-  const bootstrapDashboard = (_: string) => {
+  const createDashboardFromModel = (_: string) => {
+    // create dashboard from model
+    const newDashboardName = getName(
+      `${modelName}_dashboard`,
+      $dashboardNames.data
+    );
+    $createFileMutation.mutate(
+      {
+        data: {
+          instanceId: $runtimeStore.instanceId,
+          path: `dashboards/${newDashboardName}.yaml`,
+          blob: metricsTemplate, // TODO: compile a real yaml file
```

Comment:
`generateMeasuresAndDimension` function method can be used to compile a YAML with proper fields